### PR TITLE
Adding docs about Grafana upgrade

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-4118

### CODE REVIEW

File: observability/verifying_observability.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[#verifying-observability-version]
+= Verifying observability version
+
+Verify the observability version from the command line interface (CLI) or from Grafana user interface.
```
Comment (oafischer) [inline observability/verifying_observability.adoc]:
```suggestion
Verify the observability version from the command line interface (CLI) or from the Grafana user interface. 
```

File: observability/verifying_observability.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+After you log in to your hub cluster, access the `observabilty-grafana` pod terminal. Run the following command:
+
+----
+grafana-cli
+----
+
+The Grafana version that is currently deployed within the cluster environment is displayed.
+
+Alternatively, you can navigate to the _Manage_ tab in the Grafana dashboard. Scroll to the end of the page, where the version is listed
```
Comment (oafischer) [inline observability/verifying_observability.adoc]:
```suggestion
Alternatively, you can navigate to the _Manage_ tab in the Grafana dashboard. Scroll to the end of the page, where the version is listed.
```

**Review by oafischer — APPROVED** [review]
Looks good, just some small typos.

File: observability/verifying_observability.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+[#verifying-observability-version]
+= Verifying observability version
+
+Verify the observability version from the command line interface (CLI) or from the Grafana user interface. 
+
```
Comment (dislbenn) [inline observability/verifying_observability.adoc]: > Verify the observability version from the command line interface (CLI) or from the Grafana user interface.

From the Grafana user interface, you are only able to verify the Grafana version, not the Observability version. Similar to Thanos, if you execute the command `thanos --version` within the following pods, you will be able to see what Thanos version that Observability is deploying:
- observability-thanos-compact
- observability-thanos-query
- observability-thanos-query-frontend
- observability-thanos-receive-default
- observability-thanos-rule
- observability-thanos-store-shard

Comment (dislbenn) [reply observability/verifying_observability.adoc]: @dockerymick Can we add the Thanos version verification step to this section also?

**Comment by dockerymick** [comment]
@dislbenn thanks for the feedback. I made a couple of updates. When you have an opportunity, will you rereview the PR?

**Review by dislbenn — APPROVED** [review]
The changes look good to me! Thanks @dockerymick

**Comment by dockerymick** [comment]
Thanks for your review ! I will merge after I run the link check tool

**Pull request merged**